analysis.go

```diff
@@ -30,943 +20,3 @@ func init() {
 	archListString := "386 amd64 amd64p32 arm armbe arm64 arm64be ppc64 ppc64le mips mipsle mips64 mips64le mips64p32 mips64p32le ppc s390 s390x sparc sparc64"
 	archList = strings.Split(archListString, " ")
```

Comment:
@sdboyer 

I haven't found any references to these variables. Could you provide some details, why they are here?